content/sse/index.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: SSE and Raw HTTP Streaming API
+section: streaming-api
```

Comment:
Does this appear in the nav anywhere?  I can't see it in https://ably-docs-pr-666.herokuapp.com/